examples/subscriptions/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "examples-subscriptions"
+
+publish.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dev-dependencies]
+alloy-contract.workspace = true
+alloy-network.workspace = true
+alloy-node-bindings.workspace = true
+alloy-provider = { workspace = true, features = ["pubsub", "ws"] }
+alloy-pubsub.workspace = true
+alloy-primitives.workspace = true
+alloy-rpc-client.workspace = true
+alloy-rpc-types.workspace = true
+alloy-sol-types = { workspace = true }
```

Comment:
Please use the new Alloy namespace (see: https://github.com/alloy-rs/examples/pull/11)